src/components/ui/button.svelte

```diff
@@ -55,6 +57,7 @@
     {disabled}
     class="btn {classes.join(' ')}"
     on:click on:mousedown on:mouseup
+    title="{tooltip}"
```

Comment:
Title is not such useful attribute that needs real prop. When I firstly saw it, the idea was born:
You can pass props as {...$$props} but this is not so good, cause in some components (and even in button) it would lead to attribute-spamming: `<button lalka="kek" required nothing="false"` etc. So, it would be cool to filter props somehow. Try to dig into it